src/dscp.c

```diff
@@ -90,6 +90,9 @@ const char * iptos2str(int iptos);
 #ifndef IPTOS_DSCP_EF
 # define	IPTOS_DSCP_EF		0xb8
 #endif /* IPTOS_DSCP_EF */
+#ifdef IPTOS_DSCP_VA
```

Comment:
Should this line begin with `#ifndef` instead of `#ifdef`? This appears to be why this branch fails CI builds.

Reply:
Yes, looks like I goofed there.